coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/transformers/NullOrderingTransformer.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.trino.rel2trino.transformers;
+
+import org.apache.calcite.sql.SqlBasicCall;
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlNodeList;
+import org.apache.calcite.sql.SqlSelect;
+import org.apache.calcite.sql.SqlWindow;
+
+import com.linkedin.coral.common.transformers.SqlCallTransformer;
+
+import static org.apache.calcite.rel.rel2sql.SqlImplementor.*;
+
+
+/**
+ * This class implements the transformation of SqlCalls with NULLS LAST operator preceded by DESC
```

Comment:
nit: `transforms the ordering in the input SqlCall to be compatible with trino engine`. 

And you could also elaborate that there's no required override for ASC and the override for DESC involves dropping NULLS LAST since it is redundant. Let's not call out the 'avoid regression' part in the class javadoc. 

Also it would be good to add another example for SqlWindow operator or link the UT where it is utilized.